packages/prosemirror-dynamic-content/package.json

```diff
@@ -44,7 +44,7 @@
     "copyfiles": "^2.1.1",
     "jest": "^24.9.0",
     "ts-jest": "^24.0.2",
-    "typescript": "^3.6.2",
+    "typescript": "^3.8",
```

Comment:
At least they're the same version here. Ideally we should only have TS specified in dev dependencies though (although I assume CRA tree shakes and won't bundle it into the output)